styles/style.css

```diff
@@ -44,11 +44,12 @@ body{
     position: relative;
     clear: both;
     width: 100%;
+    height: 5rem;
     padding: 0.5rem 1rem;
     display: flex;
     justify-content: space-around;
     align-items: center;  
-    background:var(--primary-500);
+    background:#a5f3fc;
```

Comment:
why arent you using css var here ?

Reply:
oh didnt see that one left out. will do it right away